README.rst

```diff
@@ -298,6 +298,11 @@ Please do not report security issues in public. Send security concerns via email
 Changelog
 =========
 
+2.4.1 - 2020-12-07
```

Comment:
Do we want to increment the version for each pull request?  There's no advantage to deploying these intermediate steps, right? (Other than finding out sooner if there was some bad interaction out in the real world).

Reply:
@nedbat For this specific PR I think it's good to have a version bump.

For the LTI Deep Linking ones there's no need.